### ISSUE TITLE
LinkedIn SDK Login Issue

### ISSUE THREAD

**Issue by djosephranjith** [JeffersonFilho/react-linkedin-sdk#15]
Hi, I have installed and using react-linkedin-sdk. 
The login was working absolutely fine till yesterday. But today, I see that login is not working.

Scenario:
1. Click on 'Login With LinkedIn' button
2. On LinkedIn authorization window, enter user LinkedIn credentials and submit.
3. I observed that there is no response that is coming up, after successful authentication.
[The response includes the fields like; first name, last name....]

Tech Details:

After successful authorization, per code an API should be triggered to fetch the user details.
ttps://api.linkedin.com/v1/people/~:

This API is not getting triggered. Can you please help us to fix this problem. Thanks!

**Comment by JeffersonFilho** [comment]
Thanks for reporting, I will have a look and see if this issue is happening to me too.

**Comment by jason2li** [comment]
I'm having the same problem... any luck?

**Comment by Rkuro** [comment]
+1 also having this problem

**Comment by djosephranjith** [comment]
@JeffersonFilho,  Jeff, Did you get a chance to look into this. It would be great if this issue canbe resolved. As our applications are dependent on this. Please help to fix it!! Thank you!!

**Comment by JeffersonFilho** [comment]
True! I'm experiencing the same issue, I'm currently working on it to see what is the issue! Sorry for the delay!

**Comment by JeffersonFilho** [comment]
The problem is on `window.IN.User.authorize()` is not calling the callback function after the authorisation for some reason. Still looking to see what is the issue.

**Comment by djosephranjith** [comment]
@JeffersonFilho -  I'm really glad that you could replicate the issue and could identify the root cause. I will be awaiting to hear that issue is fixed. Again, Thankful for your prompt response on this. Please do the needful!!

**Comment by djosephranjith** [comment]
@JeffersonFilho Jeff, any luck on this issue?

**Comment by JeffersonFilho** [comment]
This looks like is an issue with Linkedin SDK itself, I will looking to see if I can migrate the package to the OAuth 2.0 since https://github.com/JeffersonFilho/react-linkedin-sdk/issues/14 mentioned it will stop working on  March 1, 2019.

**Comment by djosephranjith** [comment]
@JeffersonFilho  Okay. When can it be fixed? Any plan on this. Please update as we are awaiting for this to be fixed. Thank You!

**Comment by JeffersonFilho** [comment]
Sorry for the delay, I'm currently in holidays so is quite hard for me to find decent internet and time to work on this, I will see if I can migrate to OAuth 2.0 ASAP.

**Comment by snehawesolv** [comment]
@JeffersonFilho how are you? Any tentative plan for fix? Please update.

**Comment by JeffersonFilho** [comment]
Sorry for the delay on this, I'm back home now! I will see if I migrate everything to OAuth this week!

**Comment by JeffersonFilho** [comment]
It's seems is back working again! 

https://gyazo.com/4c88c509e5276cdd9c6aba06bc6703c1

But I will start working on moving it to OAuth2 ASAP!
